src/webgpu/api/operation/vertex_state.spec.ts

```diff
@@ -0,0 +1,348 @@
+export const description = '';
```

Comment:
We'll need tests that use the primitive restart values as real indices with non-strip topologies, to make sure those behave properly (and primitive restart is appropriately enabled/disabled) across backends. However let's leave these as a TODO for the moment and do them as a followup?

These tests will be important for https://github.com/gpuweb/gpuweb/issues/1220.